Mathlib/Algebra/Order/Monoid/OrderDual.lean

```diff
@@ -88,10 +88,7 @@ instance orderedCommMonoid [OrderedCommMonoid α] : OrderedCommMonoid αᵒᵈ :
 @[to_additive OrderDual.OrderedCancelAddCommMonoid.to_contravariantClass]
 instance OrderedCancelCommMonoid.to_contravariantClass [OrderedCancelCommMonoid α] :
     ContravariantClass αᵒᵈ αᵒᵈ HMul.hMul LE.le where
-    -- Porting note: We need to specify the implicit arguments here because of
```

Comment:
Why was this deleted? We still need to make the implicit explicit?

Reply:
@Ruben-VandeVelde suggested this - please discuss among each other; I have no opinion here :-)

Reply:
I don't think there's anything actionable about this note - even if we could drop the argument, I would probably keep it for robustness.